packages/store/src/StoreCore.sol

```diff
@@ -0,0 +1,559 @@
+    return _getStaticData(table, key, schema);
+  }
+
+  /************************************************************************
+   *
+   *    GET DATA
+   *
+   ************************************************************************/
+
+  /**
+   * Get full static data for the given table and key tuple, with the given schema
+   */
+  function _getStaticData(
+    bytes32 table,
+    bytes32[] memory key,
+    Schema schema
+  ) internal view returns (bytes memory) {
+    if (schema.staticDataLength() == 0) return new bytes(0);
+
+    // Load the data from storage
+    uint256 location = _getStaticDataLocation(table, key);
+    return Storage.load({ storagePointer: location, length: schema.staticDataLength() });
+  }
+
+  /**
+   * Get a single static field from the given table and key tuple, with the given schema
+   */
+  function _getStaticField(
+    bytes32 table,
+    bytes32[] memory key,
+    uint8 schemaIndex,
+    Schema schema
+  ) internal view returns (bytes memory) {
+    // Get the length, storage location and offset of the static field
+    SchemaType schemaType = schema.atIndex(schemaIndex);
+    uint256 dataLength = getStaticByteLength(schemaType);
+    uint256 location = uint256(_getStaticDataLocation(table, key));
+    uint256 offset = _getStaticDataOffset(schema, schemaIndex);
+
+    // Load the data from storage
+
+    return Storage.load({ storagePointer: location, length: dataLength, offset: offset });
+  }
+
+  /**
+   * Get a single dynamic field from the given table and key tuple, with the given schema
+   */
+  function _getDynamicField(
+    bytes32 table,
+    bytes32[] memory key,
+    uint8 schemaIndex,
+    Schema schema
+  ) internal view returns (bytes memory) {
+    // Get the length and storage location of the dynamic field
+    uint8 dynamicSchemaIndex = schemaIndex - schema.numStaticFields();
+    uint256 location = _getDynamicDataLocation(table, key, dynamicSchemaIndex);
+    uint256 dataLength = _loadEncodedDynamicDataLength(table, key).atIndex(dynamicSchemaIndex);
+
+    return Storage.load({ storagePointer: location, length: dataLength });
+  }
+
+  /************************************************************************
+   *
+   *    HELPER FUNCTIONS
+   *
+   ************************************************************************/
+
+  /////////////////////////////////////////////////////////////////////////
+  //    STATIC DATA
+  /////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Compute the storage location based on table id and index tuple
+   */
+  function _getStaticDataLocation(bytes32 table, bytes32[] memory key) internal pure returns (uint256) {
+    return uint256(keccak256(abi.encode(SLOT, table, key)));
+  }
+
+  /**
+   * Get storage offset for the given schema and (static length) index
+   */
+  function _getStaticDataOffset(Schema schema, uint8 schemaIndex) internal pure returns (uint256) {
+    uint256 offset = 0; // skip length
```

Comment:
the comment seems confusing, like it used to skip something but now doesn't

Reply:
indeed a leftover! Removing